Examples/Tests/Larmor/inputs.mr

```diff
@@ -2,7 +2,7 @@
 max_step = 400
 
 # number of grid points
-amr.n_cell =  32  32
+amr.n_cell =  64  64
 
 # The lo and hi ends of grids are multipliers of blocking factor
 amr.blocking_factor = 16
```

Comment:
Ah yes we should probably also increase the blocking factor then.

Reply:
@RemiLehe do you want to update this as well or is this ready for merge? :)